neofetch

```diff
@@ -1086,8 +1086,9 @@ get_cpu() {
                     deg="${deg/C}"
                 ;;
                 "OpenBSD"* | "Bitrig"*)
-                    deg="$(sysctl -n hw.sensors.lm0.temp0)"
-                    deg="${deg/ degC}"
+                    deg_var="$(sysctl hw.sensors | grep -m1 -E 'lm0.temp0|cpu0.temp0' | cut -d'=' -f1)"
```

Comment:
What's the full output of `sysctl hw.sensors`? I don't think we need to use both `grep` and `cut` here.

Reply:
This should work:

```sh
deg_var="$(sysctl hw.sensors | awk -F '=| ' '/lm0.temp|cpu0.temp/ {print $4; exit}')"

```

Reply:
```
$ sysctl hw.sensors
hw.sensors.cpu0.temp0=41.00 degC
hw.sensors.acpitz0.temp0=40.00 degC (zone temperature)
hw.sensors.acpibtn0.indicator0=On (lid open)
hw.sensors.acpibat0.volt0=11.10 VDC (voltage)
hw.sensors.acpibat0.volt1=12.20 VDC (current voltage)
hw.sensors.acpibat0.power0=0.00 W (rate)
hw.sensors.acpibat0.watthour0=20.65 Wh (last full capacity)
hw.sensors.acpibat0.watthour1=1.03 Wh (warning capacity)
hw.sensors.acpibat0.watthour2=0.20 Wh (low capacity)
hw.sensors.acpibat0.watthour3=20.65 Wh (remaining capacity), OK
hw.sensors.acpibat0.watthour4=23.20 Wh (design capacity)
hw.sensors.acpibat0.raw0=0 (battery full), OK
hw.sensors.acpibat1.volt0=11.10 VDC (voltage)
hw.sensors.acpibat1.volt1=12.34 VDC (current voltage)
hw.sensors.acpibat1.power0=0.00 W (rate)
hw.sensors.acpibat1.watthour0=18.95 Wh (last full capacity)
hw.sensors.acpibat1.watthour1=0.95 Wh (warning capacity)
hw.sensors.acpibat1.watthour2=0.20 Wh (low capacity)
hw.sensors.acpibat1.watthour3=18.93 Wh (remaining capacity), OK
hw.sensors.acpibat1.watthour4=23.20 Wh (design capacity)
hw.sensors.acpibat1.raw0=0 (battery idle), OK
hw.sensors.acpiac0.indicator0=On (power supply)
hw.sensors.acpithinkpad0.temp0=40.00 degC
hw.sensors.acpithinkpad0.temp1=40.00 degC
hw.sensors.acpithinkpad0.temp2=40.00 degC
hw.sensors.acpithinkpad0.temp3=40.00 degC
hw.sensors.acpithinkpad0.temp4=40.00 degC
hw.sensors.acpithinkpad0.temp5=40.00 degC
hw.sensors.acpithinkpad0.temp6=40.00 degC
hw.sensors.acpithinkpad0.temp7=40.00 degC
hw.sensors.acpithinkpad0.fan0=3073 RPM
```

And your awk statement works if you use $1 instead of $4.  On my machine:
```
$ sysctl hw.sensors | awk -F '=| ' '/lm0.temp|cpu0.temp/ {print $1; exit}'
hw.sensors.cpu0.temp0
```

You might be conflating deg_var (which I used to represent the sysctl variable name) and deg, which is the actual value that sysctl variable returns.

Reply:
Or use $2 to get the actual value.  Not sure which you were aiming for, I guess.

Reply:
Yeah, I meant `$deg`. 

The `awk` statement above was based on this:

```
  hw.sensors.lm0.temp0=26.00 degC
  hw.sensors.lm0.temp1=31.00 degC
  hw.sensors.lm0.fan1=1607 RPM
  hw.sensors.lm0.fan2=1308 RPM
  hw.sensors.lm0.volt0=1.10 VDC (VCore)
  hw.sensors.lm0.volt1=11.31 VDC (+12V)
  hw.sensors.lm0.volt2=3.23 VDC (+3.3V)
  hw.sensors.lm0.volt3=4.85 VDC (+5V)
  hw.sensors.lm0.volt4=6.07 VDC (-12V)
  hw.sensors.lm0.volt5=5.12 VDC (5VSB)
  hw.sensors.lm0.volt6=0.58 VDC (VBAT)
```

It's meant to get the value directly.

Reply:
Using both your output and mine, this works on both:

```sh
sysctl hw.sensors | awk -F '=| degC' '/lm0.temp|cpu0.temp/ {print $2; exit}'
```

Reply:
That seems to work perfectly, combined with the `deg="${deg/00/0}"`